bindings/matrix-sdk-ffi/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to this project will be documented in this file.
 
 Breaking changes:
 
+- `Client::reset_server_capabilities` has been renamed to `Client::reset_server_info`.
+  ([#5167](https://github.com/matrix-org/matrix-rust-sdk/pull/5167))
```

Comment:
Well that's still in the wrong place, this PR is not part of the 0.12.0 release from two weeks ago.

Reply:
Ohhhh sorry I see what you're saying, I conflated the ordering comment with the "wrong place" comments… 🙈